packages/admin/src/pages/dashboard/index.page.tsx

```diff
@@ -20,13 +21,14 @@ export const getServerSideProps = async ({
   query,
 }: GetServerSidePropsContext) => {
   const paginationParams: Pagination = {
-    paginate: true,
+    paginate: false,
     page: 0,
     size: 2,
-    sort: 'createdDate,DESC',
+    sort: 'lastUpdated,DESC',
   };
   const sessionCookie = getSessionIdFromCookies(req);
-  const schemes = await getUserSchemes(paginationParams, sessionCookie);
+  const { ownedSchemes: schemesUserOwns, editableSchemes: schemesUserCanEdit } =
```

Comment:
I still think these are being renamed unnecessarily. We could just pass in what the service gives us